libmozevent/bus.py

```diff
@@ -53,7 +53,8 @@ async def send(self, name: str, payload: Any):
         queue = self.queues[name]
 
         if isinstance(queue, RedisQueue):
-            async with AsyncRedis() as redis:
+            redis = await AsyncRedis.connect()
+            if redis is not None:
```

Comment:
We shouldn't silently do nothing when we fail to connect, let's turn this into an assertion: `assert redis is not None`

Reply:
(same in the following instances of this in the patch)